extension/docs/DeploymentGuide.md

```diff
@@ -29,7 +29,7 @@ Following environment variables must be provided in order to run the extension m
 | --- | --- | --- | --- |
 |`ADYEN_MERCHANT_ACCOUNT` | The name of your merchant account. | YES | |
 |`ADYEN_API_KEY` | You'll be making API requests that are authenticated with an [API key](https://docs.adyen.com/user-management/how-to-get-the-api-key#page-introduction). | YES | |
-|`ADYEN_API_BASE_URL` | [Checkout endpoint](https://docs.adyen.com/development-resources/live-endpoints#checkout-endpoints) of Adyen. | YES | `https://checkout-test.adyen.com/v52` |
+|`ADYEN_API_BASE_URL` | [Checkout endpoint](https://docs.adyen.com/development-resources/live-endpoints#checkout-endpoints) of Adyen. | YES | `https://checkout-test.adyen.com/v65` |
 |`ADYEN_LEGACY_API_BASE_URL` | [Standart payment endpoint](https://docs.adyen.com/development-resources/live-endpoints#standard-payments-endpoints) of Adyen. | YES | `https://pal-test.adyen.com/pal/servlet/Payment/v52` |
```

Comment:
Do we need to update this 'pal-test' newest version ?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/469/commits/b9261317010b95bcc445fb48627fc5c6b5f5ffcf

Reply:
64 or 65 ? 🤔

Reply:
It's a trick: https://docs.adyen.com/api-explorer/#/Payment/v64/overview

Reply:
Another trick: the JSON response has changed its structure in the newest version at least for `getPaymentMethodsResponse`. Since this is not the part of the task and there might be bigger changes, I reverted it.